docs/book/orders/checkout.rst

```diff
@@ -93,6 +93,21 @@ Firstly if the **Customer** is not yet set on the Order it will be assigned depe
             </class>
         </constraint-mapping>
 
+    If you want this constraint on API side you need to add this:
```

Comment:
If I see correctly, you prepared a validator, but we are not using it anywhere, it looks like a pretty dead code :(
WDYT about adding to this doc also snippet with validator code?

Reply:
Or maybe we should transform this note to be a separate cookbook and add here only a reference to it 🤔